Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: consul
-version: 0.1.0
+version: 0.3.0
```

Comment:
We aren't doing minor version bumps?

Reply:
We will be doing versioning as expected moving forward. I think this was overlooked when we released 0.2.0 and 0.3.0, so I'm updating it to bring it in line with what currently exists. Moving forward, this should get updated every time we release.